src/src/rx_main.cpp

```diff
@@ -495,10 +495,7 @@ bool ICACHE_RAM_ATTR HandleSendTelemetryResponse()
     alreadyTLMresp = true;
     otaPkt.std.type = PACKET_TYPE_TLM;
 
-    bool noAirportDataQueued = firmwareOptions.is_airport && apOutputBuffer.size() == 0;
-    bool noTlmQueued = !TelemetrySender.IsActive() && noAirportDataQueued;
-
-    if (NextTelemetryType == ELRS_TELEMETRY_TYPE_LINK || noTlmQueued)
+    if (NextTelemetryType == ELRS_TELEMETRY_TYPE_LINK)
```

Comment:
This looks like a regression to me.
The previous code sent link stats packets if there is no telem data queued.
This new change will only send a link stats pkt every 2nd telem slot if no data is queued.

Reply:
Nice find on catching the incorrect AP buffer!